File: CrazyBooks.Web/wwwroot/Scripts/Models/Book.js
class Book extends Entity
{
    get ComposedName()
    {
        return this.Title + ", " + this.Author;
    }

    constructor(json)
    {
        super( json);

        if (json)
        {
            this.Title = json.title;
            this.Author = json.author;
            this.PublicationDate = json.publicationDate;
            this.Edition = json.edition;
        }
        else
        {
            this.Title = "";
            this.Author = "";
            this.PublicationDate = null;
            this.Edition = 0;
        }
    }
}

File: CrazyBooks.Lib/Models/Book.cs
using System;
using System.Collections.Generic;
using System.Linq;
using Newtonsoft.Json;
using CrazyBooks.Lib.Core;

namespace CrazyBooks.Lib.Models
{
    public class Book : Entity
    {
        public string Title { get; set; }

        public string Author { get; set; }

        public int Year { get; set; }

        public int Edition { get; set; }

        [JsonIgnore]
        public ICollection<Lend> Lends { get; set; }

        public List<Guid> LendsIds
        {
            get
            {
                //if (Lends == null)
                //    return new List<Guid>();
                //else
                //    return Lends.Select(x => x.Id).ToList();

                // esto es lo mismo que lo de arriba pero con el operador ternario
                return Lends == null ? new List<Guid>() : Lends.Select(x => x.Id).ToList();
            }
        }
    }
}
